src/api/internal.rs

```diff
@@ -433,6 +437,15 @@ impl<T: Pixel> ContextInner<T> {
     Ok(())
   }
 
+  /*
+  #[cfg(feature = "dump_lookahead_data")]
```

Comment:
Why this?

Reply:
I had to reintroduce this, since L598
```diff
    };
+     #[cfg(feature = "dump_lookahead_data")]
+     let data_location =
+       Self::build_dump_properties().as_path().display().to_string();
```

without the flag, clippy says variable is not used and advises to remove if we keep that as only optional then the function is warned as not used, so kept flag, I think it will reduce the overhead marginally and users/devs who do not use this feature, do not need this.

Reply:
I have used `#[allow(unused)]` to fix this